web_client/js/imageViewerWidget/slideatlas.js

```diff
@@ -1,37 +1,52 @@
-girder.views.SlideatlasImageViewerWidget = girder.views.ImageViewerWidget.extend({
+girder.views.SlideAtlasImageViewerWidget = girder.views.ImageViewerWidget.extend({
     initialize: function (settings) {
         girder.views.ImageViewerWidget.prototype.initialize.call(this, settings);
 
+        $('head').prepend(
+            $('<link rel="stylesheet" href="http://beta.slide-atlas.org/webgl-viewer/static/css/sa.css">'));
+
         $.getScript(
-            'slideatlas.js',
+            'https://beta.slide-atlas.org/webgl-viewer/static/sa.max.js',
             _.bind(function () {
                 this.render();
             }, this)
         );
-
     },
 
     render: function () {
         // If script or metadata isn't loaded, then abort
-        if (!window.slideatlas || !this.tileSize) {
+        if (!window.SlideAtlas || !this.tileSize) {
             return;
         }
 
         // TODO: if a viewer already exists, do we render again?
-
-        this.viewer = new slideatlas();
+        $(this.el).saViewer({
```

Comment:
You should use ``this.$el`` instead of ``$(this.el)``.

See: http://backbonejs.org/#View-$el